### DISCUSSION TITLE
Once again about the inconstancy of the Hubble parameter

### DISCUSSION THREAD

**Discussion by VitalyPetrikov** [a-cosmology-group/acg#211] [Cosmology Models]
I previously wrote about the variability of the Hubble parameter here: 
https://github.com/orgs/a-cosmology-group/discussions/115

I draw your attention to the recent MailOnline publication at https://www.dailymail.co.uk/sciencetech/article-10844775/Hubble-detects-changes-universes-rate-expansion-explained-current-physics.html published 23.05.2022

It reports that NASA has been conducted research using Hubble Telescope for 30 years "to more accurately measure the rate of Universe expansion.
As a result, 40 galaxies ("milepost markers") were determined, and the distance to them and their speed are accurately measured. Distances were obtained from Cepheids and Type Ia supernovae, and velocities were obtained from the redshift of their spectrum. In order to fully ensure the reliability of their research, Hubble astronomers did not even use the last step of the "cosmic distance ladder," namely the range of "tired light." Therefore, their results are beyond any doubt.
The results shocked them. It turned out that for nearby galaxies the Hubble parameter is 73 km per megaparsec, and in the “depths of space” (but not further than reliable distance measurements) - 67.5 km per megaparsec. To the credit of the Hubble astronomers, it should be noted that they did not even hint in favor of expanding space. For them, the explored space is flat, three-dimensional. "NASA said astronomers are 'at a loss', unable to understand why there are two different values, but suggested we may have to rethink basic physics." There is no need to revise the basics of physics. I have long argued on our pages that the Hubble parameter is not a constant. And the further into the depths of space, the smaller its value will be, approaching zero at velocities of galaxy expansion close to the light speed. The matter can be continued with JWST, which is capable of finding Type Ia supernovae even in the most distant galaxies. At the same time, naturally, the space of the entire cosmos is flat three-dimensional and does not expand anywhere.

More details about inconstancy of Hubble parameter: 
[VVPetrikov About recession of galaxies 2nd edition.pdf](https://github.com/a-cosmology-group/acg/files/14990314/VVPetrikov.About.recession.of.galaxies.2nd.edition.pdf)

**Comment by mikehelland** [comment]
I'd just like to clarify a couple things.

>The results shocked them. It turned out that for nearby galaxies the Hubble parameter is 73 km per megaparsec, 

They say "nearby" and "local", but that's very misleading.

The SN1a data goes out z=2.26, which, in LCDM, is a lookback time of 10.6 billion years, or ~77% of the way back to the big bang.

![tz-local](https://github.com/a-cosmology-group/acg/assets/19311107/dbe5e146-e378-47db-8d50-1c65fcc72685)


>and in the “depths of space” (but not further than reliable distance measurements) - 67.5 km per megaparsec. 

This value is based on measurements of the CMB, at about z = 1100, so yes, much further than reliable distance measurements.


>I have long argued on our pages that the Hubble parameter is not a constant. 

In mainstream LCDM cosmology $H$ is Hubble's parameter, and $H_0$ is Hubble's constant, where.

$$H = H_0 [\Omega_m (1+z)^3 + \Omega_\Lambda]^{1/2} $$

So even in the mainstream, Hubble's parameter is not constant. Hubble's constant is just the value of Hubble's parameter today.

**Reply by VitalyPetrikov** [reply]
Dear Mike, are you questioning the 30-year work of a team of NASA astronomers?

I think that the words "nearby" and "local" express the distances available to Edwin Hubble in his time. At these distances, NASA astronomers searched for the most accurate value of the Hubble parameter. To do this, they used several galaxies (available to E. Hubble) from the 40 most reliable galaxies. And they placed them like that.

And the expression “in the depths of space” means a distance somewhat further than E. Hubble could see, but not so far where various modern cosmological speculations are already possible. This is why the parameters of these 40 selected galaxies are extremely reliable.

> In mainstream LCDM cosmology

 
at one time, geocentric astronomy also dominated, as did, for example, the theory of a special substance during the transfer of heat.

> Hubble's constant is just the value of Hubble's parameter today.

precisely today the Hubble parameter in the entire range of its manifestation is not a constant value, much less a constant. And the fact that it changes over time is self-evident, because galaxies move away consistently while maintaining their speeds. It’s just that 100–200 years doesn’t change anything for space.
